src/resources/scss/bioTheme/layouts/_grid.scss

```diff
@@ -1,6 +1,6 @@
 @import "../utilities/functions";
-@import "../../../../node_modules/breakpoint-sass/stylesheets/breakpoint";
```

Comment:
I'm not sure if we should adapt the paths like this or better let them relative and add just one ../ more instead of starting at root. what do you guys think?

Reply:
I think we better let them relative